canonical/charmhub.io
# update cards to the same as integration cards from store components

## Done
- updated cards on the integration tab to the same as integration cards from store components

## How to QA
- go to https://charmhub-io-1745.demos.haus/traefik-k8s/integrations
- compare cards from [integration cards from store components](https://canonical.github.io/store-components/?path=/docs/integrationcard--docs)
- check `search and filter` works as expected

## Issue / Card
Fixes #https://warthogs.atlassian.net/browse/WD-4143
Fixes #https://warthogs.atlassian.net/browse/WD-4940
